test/credits/canonical_names_test.rb

```diff
@@ -1460,7 +1460,7 @@ class CanonicalNamesTest < ActiveSupport::TestCase
     end
 
     test "evgeny.zislis\100gmail.com" do
-      assert_contributor_names '842ce34', 'Evgeny Zislis'
+      assert_contributor_names '842ce34bbcfb2d65b04a6e80bf8c168d7c17277d', 'Evgeny Zislis'
```

Comment:
**polish:** Based on the conversation at Rails World, I'm shortening the SHA to achieve the disambiguation *and* improved developer aesthetics.

```suggestion
      assert_contributor_names '842ce34b', 'Evgeny Zislis'
```